source/_components/alarmdecoder.markdown

```diff
@@ -40,6 +40,7 @@ alarmdecoder:
       name: 'Smoke Detector'
       type: 'smoke'
       rfid: '0123456'
+      loop: 1
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

```suggestion
```

Reply:
No problem. :-)